### PR TITLE
Feat: Performance and Monitoring bubble

### PR DESCRIPTION
**Pull request by lukasz-lapczyk**
https://github.com/Selleo/DevPath/issues/82

### CODE REVIEW

File: content/web_development/skills/performance_and_monitoring.md [hunk 1]
```diff
@@ -12,30 +12,125 @@ title = "Performance and monitoring"
 
 **Person which successfully completed requirement for given block can:** 
 
-- Has strong and working knowledge in setting up and leveraging a wide range of performance monitoring tools
+- Has strong and working knowledge in setting up and leveraging a wide range of performance monitoring and error tracking tools
 - Is capable of precisesly identifying reasons behind performance problems in web applications
-- Has strong skillset in using JMeter to prepare peak-load and volume testing scenarios and is capable of determining maximum throughput for given environment
+- Has strong skillset in using load testing tools to prepare peak-load and volume testing scenarios and is capable of determining maximum throughput for given environment
```
Comment (PoslinskiNet) [inline content/web_development/skills/performance_and_monitoring.md:17]: 👍🏼

File: content/web_development/skills/performance_and_monitoring.md [hunk 2]
```diff
@@ -12,30 +12,125 @@ title = "Performance and monitoring"
...
+- Set up preferred open source tool and integrate it in your app.
+
+## 📦 Performance / analysis
+
+### 📦 Profiling
+
+#### 🎤 Learn
+
+- [How do Ruby/Python profilers work](https://jvns.ca/blog/2017/12/17/how-do-ruby---python-profilers-work-/)
+- [Understanding ruby app with profiling](https://katafrakt.me/2020/05/03/understanding-ruby-app-with-profiling/)
```
Comment (PoslinskiNet) [inline content/web_development/skills/performance_and_monitoring.md:81]: @mikoscz can you add Elixir solutions here?
@simon2k can you add Node/Nest solutions here?

File: content/web_development/skills/performance_and_monitoring.md [hunk 3]
```diff
@@ -12,30 +12,125 @@ title = "Performance and monitoring"
...
+
+#### 📝 Katas
+
+- Profile a part of your app, visualise and explain the results (e.g. on flamegraph)
+
+### 📦 Benchmarking
+
+#### 🎤 Learn
+
+- [Benchmarking Ruby code #1](https://blog.appsignal.com/2018/02/27/benchmarking-ruby-code.html)
```
Comment (PoslinskiNet) [inline content/web_development/skills/performance_and_monitoring.md:105]: @simon2k can you add Node/Nest solutions here?

Comment (PoslinskiNet) [reply content/web_development/skills/performance_and_monitoring.md:105]: @lukasz-lapczyk maybe we could add some external benchmarking here that is language-agnostic that works externally?

File: content/web_development/skills/performance_and_monitoring.md [hunk 4]
```diff
@@ -12,30 +12,125 @@ title = "Performance and monitoring"
...
+
+#### 📝 Katas
+
+- Benchmark a piece of code and explain the results
+
+### 📦 Garbage collection
+
+#### 🎤 Learn
+
+- [Garbage collection fundamentals](https://docs.microsoft.com/en-us/dotnet/standard/garbage-collection/fundamentals)
```
Comment (PoslinskiNet) [inline content/web_development/skills/performance_and_monitoring.md:122]: @simon2k would be nice to have things around NodeJS/NestJS (datadog for instance, pm2 etc).
@mikoscz is that a huge deal in Erlang/Elixir world? If so, please provide some info around it.

File: content/web_development/skills/performance_and_monitoring.md [hunk 5]
```diff
@@ -12,30 +12,125 @@ title = "Performance and monitoring"
...
+- Has strong skillset in using load testing tools to prepare peak-load and volume testing scenarios and is capable of determining maximum throughput for given environment
 - Is capable of recommending right infrastructure configuration to meet performance needs
 
 {{% /bubble%}}
 
 {{%todo %}}
+- **Frontend part**
+- NodeJS - add tools, learning materials, interview questions, katas
+- Elixir - verify existing materials and provide some more
+{{% /todo%}}
```
Comment (PoslinskiNet) [inline content/web_development/skills/performance_and_monitoring.md:26]: ✂️  @lukasz-lapczyk

File: content/web_development/skills/performance_and_monitoring.md (hunk 2 above)
Comment (mikoscz) [reply content/web_development/skills/performance_and_monitoring.md:89]: done

File: content/web_development/skills/performance_and_monitoring.md (hunk 4 above)
Comment (mikoscz) [reply content/web_development/skills/performance_and_monitoring.md:131]: done

File: content/web_development/skills/performance_and_monitoring.md [hunk 6]
```diff
@@ -6,36 +6,141 @@ title = "Performance and monitoring"
...
+- [Datadog](https://www.datadoghq.com/)
+
+**Open source:**
+
+- [Prometheus](https://prometheus.io/)
+- [Graphite](https://graphiteapp.org/)
+- [Zabbix](https://www.zabbix.com/)
+- [Grafana](https://grafana.com/) (visualisation)
+
+**Browser**
```
Comment (lukasz-lapczyk) [inline content/web_development/skills/performance_and_monitoring.md:55]: @PoslinskiNet Shouldn't this section be called **Error tracking**?

File: content/web_development/skills/performance_and_monitoring.md (hunk 3 above)
Comment (lukasz-lapczyk) [reply content/web_development/skills/performance_and_monitoring.md:114]: Unfortunately I haven't found any, as benchmarking is very language-specific topic.

**Pull request merged**